### PR TITLE
chore(common): Add crowdin strings for Kannada 🍒 

### LINKED ISSUE #7952: chore(common): Request Kannada localization
### Describe the bug

Kannada localization is completed in crowdin.

Please pull it into all apps. I would like to test it.

### Reproduce the bug

_No response_

### Expected behavior

_No response_

### Related issues

_No response_

### Keyman apps

- [x] Keyman for Android
- [x] Keyman for iPhone and iPad
- [x] Keyman for Linux
- [x] Keyman for macOS
- [x] Keyman for Windows
- [ ] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

_No response_

### Operating system

_No response_

### Device

_No response_

### Target application

_No response_

### Browser

_No response_

### Keyboard name

_No response_

### Keyboard version

_No response_

### Language name

Kannada

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by sgschantz**
🍒-pick of #7970 to fix #7952 in 17-alpha

This adds the crowdin strings for Kannada - locale kn-IN

## User Testing
Load the PR build for each platform and set the UI / Locale to "ಕನ್ನಡ" (and/or Kannada)

* **TEST_ANDROID** 
1. Load the PR build of Keyman for Android
2. In the Keyman app, go to Settings --> Display Language --> ಕನ್ನಡ (Kannada)
3. Return to the Keyman app 
4. Verify the UI is in Kannada (prompt, Settings strings, etc.). Note, the globe tip string stays in English and is a separate issue #7975

kannada android menu

* **TEST_IOS**
1. Load the PR build of Keyman for iPhone and iPad
2. Set the language to Kannada
3. Return to the Keyman app
4. Verify the UI is in  Kannada

* **TEST_LINUX**
1. Start Keyman configuration with: LANGUAGE=kn_IN km-config
2. Verify Keyman UI is in Kannada

* **TEST_MACOS**
1. Load the PR build of Keyman for macOS
2. Set the language to Kannada
3. Verify the Keyman UI is in Kannada

* **TEST_WINDOWS**
1. Load the PR build of Keyman for Windows
2. Set the UI to Kannada
3. Verify the Keyman UI is in Kannada

### CODE REVIEW

File: ios/keyman/Keyman/Keyman.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1116,9 +1119,11 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Keyman/Keyman.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
```
Comment (mcdurdin) [inline ios/keyman/Keyman/Keyman.xcodeproj/project.pbxproj:1122]: The additions in this .pbxproj may be causing the build to fail?

**Comment by bharanidharanj** [comment]
* **TEST_IOS (PASSED):** Tested with the attached PR build in iPhone and iPad (iOS 15.2) Simulators and it is working as expected. ie., UI language of the Keyman has been changed into Kannada. 

..Kannada UI in Keyman App (iPhone)

   

..Kannada UI in Keyman App (iPad)

**Comment by bharanidharanj** [comment]
* **TEST_WINDOWS (PASSED):** Tested with the attached PR build in Windows 11 (VM) and I noticed that the Keyman UI has been changed into Kannada in the Configuration dialog. 

..Kannada UI1

   

..Kannada UI2

   

..Kannada UI3

   

..Kannada UI4

**Comment by bharanidharanj** [comment]
* **TEST_ANDROID (PASSED):** Tested with the attached PR build (17.0.34-alpha-test-8074) in Android 12 / API 31 emulator and the UI language appears in Kannada after changing the display language into Kannada.

**Comment by bharanidharanj** [comment]
# Test Results

* **TEST_MACOS (PASSED):** Tested with the attached PR build in mac Mini OS (Monterey 12.6) and I was able to see the UI language has been changed into Kannada in the most of the views. 

..Kannada UI1:

   


..Kannada UI2:

**Review by rc-swag — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.36-alpha](https://keyman.com/downloads/releases/alpha/17.0.36)

**Pull request merged**